### PR TITLE
WIP: Non-legacy ResourcesShortcutStatic

### PR DESCRIPTION
**Pull request by sttts**
/cc @mfojtik

### CODE REVIEW

**Comment by sttts** [comment]
[test]

File: vendor/k8s.io/kubernetes/pkg/kubectl/kubectl.go [hunk 1]
```diff
@@ -138,34 +138,33 @@ var ResourcesShortcutStatic = []ResourceShortcuts{
...
 	},
-
 	{
 		ShortForm: schema.GroupResource{Resource: "scc"},
 		LongForm:  schema.GroupResource{Resource: "securityContextConstraints"},
 	},
 	{
-		ShortForm: schema.GroupResource{Resource: "dc"},
-		LongForm:  schema.GroupResource{Resource: "deploymentConfigs"},
+		ShortForm: schema.GroupResource{Group: "deploy.openshift.io", Resource: "dc"},
```
Comment (mfojtik) [inline vendor/k8s.io/kubernetes/pkg/kubectl/kubectl.go]: apps.openshift.io

Comment (mfojtik) [inline vendor/k8s.io/kubernetes/pkg/kubectl/kubectl.go]: what is interesting is that `oc get dc` still works fine ;-)

**Comment by sttts** [comment]
[test]

**Comment by sttts** [comment]
[test]

**Pull request closed without merging**